.github/workflows/cypress-prod-nightly.yml

```diff
@@ -51,4 +51,4 @@ jobs:
         with:
           job-status: failure
           slack-bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
-          slack-channel: ${{ secrets.SLACK_CHANNEL }}          
+          slack-channel: ${{ secrets.SLACK_CHANNEL }}
```

Comment:
can we ensure all developers have same `linting` configuration in their workspace?

Reply:
Posted my config and extensions in slack